modules/ocf/manifests/repackage.pp

```diff
@@ -17,16 +17,18 @@
     # including the original upgrade, but then a package and all its
     # dependencies would need to be pinned, which is excessive and prone to
     # breaking if a package's dependencies change.
-    exec { "/usr/bin/apt-get -y -o Dpkg::Options::=--force-confold ${install_options[0]} -t ${dist} install ${package}":
+    exec { "apt install ${package}":
+      command => "/usr/bin/apt-get -y -o Dpkg::Options::=--force-confold ${install_options[0]} -t ${dist} install ${package}",
       logoutput   => on_failure,
       environment => [
         'DEBIAN_FRONTEND=noninteractive',
       ],
       unless      => "dpkg-query -W ${package} | grep \\~bpo";
+      before      => Package[$package];
     }
-  } else {
-    package { $package:
-      install_options => $install_options;
-    }
+  }
+
+  package { $package:
+    install_options => $install_options;
```

Comment:
That's really nice, this has been mildly annoying for a while, so this will be much better!